### PR TITLE
323 Applying solid colors to the cards

### PR DESCRIPTION
**Pull request by sonaldhekale**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--cn-website--netcentric.hlx.page/
- After: https://323-implement-solid-colors--cn-website--netcentric.hlx.page/drafts/sdhekale/cards-solidcolors

### CODE REVIEW

**Comment by sonaldhekale** [comment]
Hello @alexnerdrumhansen i cleaned the PR so you can test it on https://323-implement-solid-colors--cn-website--netcentric.hlx.page/drafts/sdhekale/cards-solidcolors

**Review by nc-andreashaller — APPROVED** [review]

**Comment by alexnerdrumhansen** [comment]
The Teal colour on the left should be Midnight Blue as seen here: 
https://www.figma.com/file/OeE1zEoK84ckQpsAVMP6Ps/Modifications-(Copy)?type=design&node-id=1060-13437&t=e5Pt6puL43AZNAUw-0

**Comment by DaphneKoren** [comment]
issue is fixed both block options colour and images look good 
Screenshot 2023-05-25 at 11 43 03

File: blocks/cards/cards.css [hunk 1]
```diff
@@ -421,3 +430,56 @@
   background-color: var(--bg-text-light-plum);
   color: var(--c-dark-plum);
 }
+
+:root {
+    --c-midnight-blue: #000048;
```
Comment (nc-andreashaller) [inline blocks/cards/cards.css:435]: as far as I can see we have this already defined: https://github.com/Netcentric/cn-website/blob/main/styles/styles.css#L43

**Pull request merged**